sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -601,7 +601,7 @@ public void testTableConstraintsWithOrderBy() {
     @Test
     public void testDefaultColumns() {
         execute("select * from information_schema.columns order by table_schema, table_name");
-        assertEquals(772, response.rowCount());
+        assertEquals(774, response.rowCount());
```

Comment:
just curious: what caused this change?

Reply:
`settings.translog.durability` and `settings.scheduler.max_merge_count` were returned by the expressions but not defined in the schema.